### DISCUSSION TITLE
ScrollArea with a Grid with static header row

### DISCUSSION THREAD

**Discussion by Dapeos** [emilk/egui#776] [Q&A]
I am using a grid inside a ScrollArea to save on performance as the tables I have contain thousands of records. The issue is that when I scroll I lose my column headers. 

I tried wrapping the ScrollArea with the Grid after defining the Header row but the ScrollArea::from_max_height(f32::INFINITY) then reports 0. 

I can define two Grids one outside the ScrollArea for the header row and one inside the scrollArea for the records but then I would have to use static column widths for both tables so they line up properly. 

Thinking I might just be missing something.
Thanks,

**Comment by rminderhoud** [comment]
I was looking to do this today and was curious if you figured out a solution?

**Comment by emilk** [comment]
This is now possible, by using `egui_extras::Table` (unreleased) from https://github.com/emilk/egui/pull/963

**Reply by rminderhoud** [reply]
Wonderful!!
